src/frameworks/constants.ts

```diff
@@ -47,28 +45,6 @@ export const ALLOWED_SSR_REGIONS = [
 
 export const I18N_ROOT = "/";
 
-export const WebFrameworks: Record<string, Framework> = Object.fromEntries(
```

Comment:
Turns out we were pulling in frameworks from here still, due to a bad merge. We should pull from the new frameworks.ts instead.